src/functions.c

```diff
@@ -438,10 +438,39 @@ NULL };
     addReplyHelp(c, help);
 }
 
+/* Verify that the function name is of the format: [a-zA-Z0-9_][a-zA-Z0-9_]? */
+static int functionsVerifyName(sds name) {
+    if (sdslen(name) == 0) {
+        return C_ERR;
+    }
+    for (size_t i = 0 ; i < sdslen(name) ; ++i) {
+        char curr_char = name[i];
+        if (curr_char >= 'a' && curr_char <= 'z') {
+            continue;
+        }
+        if (curr_char >= 'A' && curr_char <= 'Z') {
+            continue;
+        }
+        if (curr_char >= '0' && curr_char <= '9') {
+            continue;
+        }
+        if (curr_char == '_') {
+            continue;
+        }
```

Comment:
i hate lines 8-), and there are too many here.
two things i would suggest (take either).
1. put the `continue` in the same line as the `if`, and avoid the braces. i generally don't like to do that on statements that change the execution code (return, break, continue), but if there are multiple lines doing the same (like error handling), i think it's ok.
2. unify all these `if`s into one `if` with multiple condition lines (which is ok since the action is the same)

Reply:
Fixed